File: medialuno_pascal/medialuno.pas
program medialuno;
var
media,n1,n2,n3: real;
begin
write('Digite a primeira nota: '); 
read(n1);
write('Digite a segunda nota: '); 
read(n2);
write('Digite a terceira nota: '); 
read(n3);

media:= (n1+n2+n3)/3;

if (media>=6) then
begin 
writeln('Sua média foi: ',media,' Aprovado!');
end
else
begin
writeln('Sua média foi: ',media,' Reprovado :(');
end;
end.

File: medialuno_c#/medialuno.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Digite a primeira nota: ");
        double n1 = Convert.ToDouble(Console.ReadLine());
        
        Console.Write("Digite a segunda nota: ");
        double n2 = Convert.ToDouble(Console.ReadLine());

        Console.Write("Digite a terceira nota: ");
        double n3 = Convert.ToDouble(Console.ReadLine());
        

        double media = (n1+n2+n3) / 3.0;
        
        Console.WriteLine("Sua média é: " + media);
        if (media >= 6.0)
        {
            Console.WriteLine("Aprovado!");
        }
        else
        {
            Console.WriteLine("Reprovado :(");
        }
    }
}
